Tests/UnitTests/Purchasing/Purchases/TransactionPosterTests.swift

```diff
@@ -127,6 +127,50 @@ class TransactionPosterTests: TestCase {
         expect(self.mockTransaction.finishInvoked) == true
     }
 
+    func testHandlePurchasedTransactionSendsSK2Receipt() throws {
+        try AvailabilityChecks.iOS15APIAvailableOrSkipTest()
+
+        self.setUp(observerMode: false, usesStoreKit2JWS: true)
+        let jwsRepresentation = UUID().uuidString
+        self.mockTransaction = MockStoreTransaction(jwsRepresentation: jwsRepresentation, environment: .xcode)
+
+        let product = MockSK1Product(mockProductIdentifier: "product")
+
+        let transactionData = PurchasedTransactionData(
+            appUserID: "user",
+            source: .init(isRestore: false, initiationSource: .queue)
+        )
+
+        let receipt = StoreKit2Receipt(
+            environment: .xcode,
+            subscriptionStatusBySubscriptionGroupId: [
+                "123_subscription_group_id": [
+                    .init(state: .subscribed,
+                          renewalInfoJWSToken: "123_renewal_info_jws_token",
+                          transactionJWSToken: "123_transaction_jws_token")
+                ]
+            ],
+            transactions: ["123_transaction_jws_token"],
+            bundleId: "123_bundle_id",
+            originalApplicationVersion: "123_original_application_version",
+            originalPurchaseDate: Date(timeIntervalSince1970: 123))
+
+        self.receiptFetcher.shouldReturnReceipt = false
+        self.transactionFetcher.stubbedReceipt = receipt
+        self.productsManager.stubbedProductsCompletionResult = .success([StoreProduct(sk1Product: product)])
+        self.backend.stubbedPostReceiptResult = .success(Self.mockCustomerInfo)
+
+        let result = try self.handleTransaction(transactionData)
+        expect(result).to(beSuccess())
+        expect(result.value) === Self.mockCustomerInfo
+
+        expect(self.backend.invokedPostReceiptData) == true
+        expect(self.backend.invokedPostReceiptDataParameters?.transactionData).to(match(transactionData))
```

Comment:
Why `match`?

Reply:
Copied this bit from other tests in the file. It's a custom matcher we have:

https://github.com/RevenueCat/purchases-ios/blob/main/Tests/UnitTests/Purchasing/Purchases/TransactionPosterTests.swift#L392-L402

Reply:
Oh I forgot 🙈